ehitchhiking-ui/src/app/shared/components/rate-passengers-modal/user.ts

```diff
@@ -1,6 +1,14 @@
-export interface User {
-  id: number;
-  name: string;
-  rating: number;
-  isAddedToBlackList: boolean;
+export interface Pass {
+	idPass: number;
+	rate: number;
+}
+
+export interface Driver {
+	idDriver: number;
+	rate: number;
+}
+
+export interface BlacklistUser {
```

Comment:
bad naming: BlacklistUser . Pls rename to  BlacklistedUser

Reply:
done